### PR TITLE
Snipe Spots and Pokemon + Rename optimized -> Update Settings.cs

### PR DESCRIPTION
**Pull request by tr0n0s**

### CODE REVIEW

**Comment by kaeus** [comment]
Why are we adding sniping spots in the default config? If anything, these should be added to WIKI, not in the config.

File: PoGo.NecroBot.Logic/Settings.cs [hunk 1]
```diff
@@ -108,7 +108,7 @@ public class GlobalSettings
         public bool KeepPokemonsThatCanEvolve = false;
         public bool PrioritizeIvOverCp = true;
         public bool RenameAboveIv = true;
-        public string RenameTemplate = "{0}_{1}";
+        public string RenameTemplate = "{1}_{0}";
```
Comment (timelf123) [inline PoGo.NecroBot.Logic/Settings.cs]: "rename optimized"

**Comment by Longpc** [comment]
Please add end of snipping pokemon when we lost all pokeBall, greateBall or superball.
Current BOT still try snipping pokemon when it not have any more ball
@tr0n0s

**Comment by kaeus** [comment]
I'll be adding this as well in a separate PR.

**Comment by tr0n0s** [comment]
you have reason, closed

**Pull request closed without merging**